tests/haddock/LibB.hs

```diff
@@ -5,17 +5,12 @@ module LibB where
 
 import LibA.A (a)
 import LibA (f)
-import LibC (mytype, LibCType)
 import TH (foo)
 
 -- | Doc for 'x' using 'f' and 'a' and 'Int'.
 x :: Int
 x = const f a
 
--- | This uses a type from an undocumented package
```

Comment:
Don't we remove a potentially useful test here?

I assume that's because we can't easily extend the stackage snapshot whith some lib which isn't on hackage but maybe we could use an existing library with no documentation instead

Reply:
We are. But the value of the test needs to be measured against the cost of migrating it. It's quite difficult.

> maybe we could use an existing library with no documentation instead

Good idea. If you can find one then we could revive this test.